test/sqlsrv/srv_013_sqlsrv_get_field.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+sqlsrv_get_field() using SQLSRV_PHPTYPE_STREAM(SQLSRV_ENC_CHAR)
+--SKIPIF--
+--FILE--
+<?php
+
+require_once("autonomous_setup.php");
+
+// Connect
+$conn = sqlsrv_connect( $serverName, $connectionInfo);
+if( !$conn ) { die( print_r( sqlsrv_errors(), true)); }
```

Comment:
Would you consider creating a helper method for error handling (die(print_r(sqlsrv_errors...))?